lib/voom/presenters/plugins/geotix_seating_chart/views/geotix_seating_chart.js

```diff
@@ -0,0 +1,62 @@
+class GeotixSeatingChart {
+  constructor(element) {
+    console.log('\tSeatingChart');
+    let data = JSON.parse(element.dataset.seatingOptions);
+    let options = {
+      divId: element.id,
+      publicKey: data.public_key,
+      event: data.event_id,
+      pricing: data.pricing.to_json,
+      availableCategories: data.available_categories,
+      priceLevelsTooltipMessage: data.price_level_tooltip
+    };
+    new seatsio.SeatingChart(options).render();
+  }
+}
+
+class GeotixSeatingDesigner {
+  constructor(element) {
+    console.log('\tSeatingDesigner');
+    let data = JSON.parse(element.dataset.designerOptions);
+    let options = {
+      divId: element.id,
+      designerKey: data.designer_key,
+      chartKey: data.chart_key,
+      language: data.language,
+      openLatestDrawing: data.open_latest_drawing,
+      features: {
+        disabled: data.disabled,
+        readOnly: data.readonly
+      }
+    };
+    if (data.chart_key_input_id) {
+      options.onChartCreated = function (chartKey) {
+        document.querySelector('#'+data.chart_key_input_id).value = chartKey;
```

Comment:
To be ready for shadow dom this will need to query from it's shadow root instead of document.  We'll need to make the `getRoot` polyfill in presenters available to plugins somehow or copy it to each
```js
function getRoot(element) {
  if (element.parentNode === null) return element;

  return getRoot(element.parentNode);
}
```
```suggestion
        getRoot(element).querySelector('#'+data.chart_key_input_id).value = chartKey;
```
Same for the 2 cases below